css/components/nav.css

```diff
@@ -0,0 +1,32 @@
+.nav{
+    display: flex;
+    flex-flow: row wrap;
+    align-items: center;
+    justify-content: space-between;
+    padding:1rem;
+    width:100%;
+    box-shadow: -1px 0px 5px 2px;
+}
+
+.start{
+  margin-right:auto;
+}
+.end{
+    margin-left:auto;
+}
+
+.side-nav{
+  box-shadow: 1px 1px 2px 2px var(--palesilver-li);
+}
+
+.side-nav>.list>li:hover{
+  font-weight: bolder;
+  cursor: pointer;
+  background-color: var(--prim-x-light);
```

Comment:
you can flex to the navbar items as it is breaking and overflowing from the div , flex wrap can you in this. Do Check!! when possible